include/swift/Basic/LangOptions.h

```diff
@@ -264,6 +264,9 @@ namespace swift {
     /// Enable features useful for running in the debugger.
     bool DebuggerSupport = false;
 
+    /// When set, the module loader will try to import private dependencies.
```

Comment:
I think it would help to add a comment that currently this is used by the debugger and we do not otherwise expose an interface to set this.

Reply:
I would avoid using the term private dependencies as private is only one of the non-public import levels, and it's also used for `@_private imports`. I would recommend instead something along the lines of `...will try to import non-public transitive dependencies` to be more exact on the behavior enabled here.

Reply:
Done.